pkg/util/workerqueue/workerqueue_test.go

```diff
@@ -194,3 +195,14 @@ func TestWorkerQueueEnqueueAfter(t *testing.T) {
 		assert.Fail(t, "should have got a queue'd message by now")
 	}
 }
+
+func TestIsDebugError(t *testing.T) {
+	err := errors.New("not a debug error")
+	assert.False(t, isDebugError(err))
+
+	err = NewDebugError(err)
+	assert.True(t, isDebugError(err))
+
+	err = NewDebugError(nil)
```

Comment:
What would isDebugError return for this case? I think it would still return true, since the type info would still wrap the nil error.

Reply:
Yep, it would, since it's a wrapper. But we should test it :+1: